core/src/main/java/org/geysermc/geyser/translator/protocol/bedrock/world/BedrockLevelSoundEventTranslator.java

```diff
@@ -66,5 +71,26 @@ public void translate(GeyserSession session, LevelSoundEventPacket packet) {
             animatePacket.setAction(AnimatePacket.Action.SWING_ARM);
             session.sendUpstreamPacket(animatePacket);
         }
+
+        // Used by client to get book from lecterns in survial mode since 1.20.70
+        if (packet.getSound() == SoundEvent.HIT) {
```

Comment:
This... cannot be right. BDS should have server-authoritative sounds and this would never be sent?

Reply:
Here's, 1:1, what BDS is sending:

```
[13:15:39:548] [SERVER BOUND] - LevelSoundEvent2Packet(sound=HIT, position=(0.5, -59.5, 2.5), extraData=127930954, identifier=, babySound=false, relativeVolumeDisabled=false)
```
which is then directly followed by
```
[13:15:39:594] [CLIENT BOUND] - LevelSoundEvent2Packet(sound=HIT, position=(0.5, -59.5, 2.5), extraData=127930954, identifier=minecraft:, babySound=false, relativeVolumeDisabled=false)
```

Reply:
Can confirm 100% this is sent by BDS.

Reply:
Well, rather, on BDS the client sends this and in survival mode this seems to be the only thing indicating that the client wants the lectern book to drop.